assets/css/_nav.scss

```diff
@@ -43,4 +43,5 @@
 .l-nav__app-content {
   grid-area: content;
   position: relative;
+  overflow: scroll;
```

Comment:
suggestion/question: could this be put on Detours List Page instead? [`overflow-auto`](https://getbootstrap.com/docs/5.3/utilities/overflow/) should scroll only if it overflows

Reply:
I need to get more comfortable with these utilities! Thanks!